warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -376,8 +376,9 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
         // preserve the original range for use with the Final Document tracking iterator because it is placed after the ResultCountingIterator
         // so the FinalDocumentTracking iterator needs the start key with the count already appended
         this.originalRange = range;
```

Comment:
separate this.waitWindowObserver.setSeekRange(range) from this.waitWindowObserver.start() since we bypass the latter call when maxYields is exceeded